QgisModelBaker/gui/dataset_manager.py

```diff
@@ -0,0 +1,258 @@
+# -*- coding: utf-8 -*-
+"""
+/***************************************************************************
+                              -------------------
+        begin                : 10.08.2021
+        git sha              : :%H$
+        copyright            : (C) 2021 by Dave Signer
+        email                : david at opengis ch
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from enum import Enum
+
+from ..libqgsprojectgen.db_factory.db_simple_factory import DbSimpleFactory
+from ..libqgsprojectgen.dbconnector.db_connector import DBConnectorError
+
+from QgisModelBaker.libili2db.globals import DbIliMode, displayDbIliMode, DbActionType
+from QgisModelBaker.libili2db.ili2dbconfig import Ili2DbCommandConfiguration
+from QgisModelBaker.gui.edit_dataset_name import EditDatasetDialog
+
+from qgis.PyQt.QtWidgets import (
+    QDialog,
+    QHeaderView,
+    QTableView,
+    QMessageBox
+)
+from qgis.PyQt.QtCore import (
+    QSettings,
+    QTimer,
+    Qt
+)
+from ..utils import get_ui_class
+
+from qgis.PyQt.QtGui import (
+    QStandardItemModel,
+    QStandardItem
+)
+
+DIALOG_UI = get_ui_class('dataset_manager.ui')
+
+class DatasetSourceModel(QStandardItemModel):
+    class Roles(Enum):
+        TID = Qt.UserRole + 1
+        DATASETNAME = Qt.UserRole + 2
+
+        def __int__(self):
+            return self.value
+
+    def __init__(self):
+        super().__init__()
+
+    def flags(self, index):
+        return Qt.ItemIsSelectable | Qt.ItemIsEnabled
+
+    def reload_datasets(self, db_connector):
```

Comment:
```suggestion
    def _reload_datasets(self, db_connector):
```

This is mostly (only?) used internally, let's prefix it with `_` in this case

Reply:
I just accepted the change, but I wasn't reading well. I thought it's about the refresh_datasets method. But this here is not called internally. It's called in `DatasetManagerDialog`... So I would not make it private.